src/App.tsx

```diff
@@ -107,10 +108,6 @@ const App = (props: AppProps) => {
   const [patchName, setPatchName] = useState(() => korgProgramDump.patchName);
   const [paramState, setParamState] = useState(() => initialiseParamState(korgProgramDump));
 
-  useEffect(() => {
```

Comment:
Turns out I didn't need this block at all, the other functions above handle this behaviour better.